app/Filament/Widgets/StatsOverviewWidget.php

```
@@ -0,0 +1,81 @@
+    protected ?string $pollingInterval = '30s';
+    protected static ?int $sort = 1;
+
+    protected function getStats(): array
+    {
+        // Statistiques des clients
+        $totalCustomers = Customer::count();
+        $activeCustomers = Customer::where('status', CustomerStatus::ACTIVE)->count();
+        $inactiveCustomers = $totalCustomers - $activeCustomers;
+
+        // Statistiques des licences
+        $totalLicenses = License::count();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$totalLicenses'. (Unused Code Rules)

(UnusedLocalVariable)